SpriterDotNet.sln.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
It isn't required, and would help contributors use the desired naming conventions etc. automatically. I don't see any harm in including this, but if you feel strongly about it, I can remove it.